### DISCUSSION TITLE
NNConv

### DISCUSSION THREAD

**Discussion by riskiem** [pyg-team/pytorch_geometric#2854] [Q&A]
In the example qm9_nn_conv.py, what does dim=64 signify?

**Comment by wsad1** [comment]
`dim=64`, is the hidden dimension used through out the  network [Net](https://github.com/rusty1s/pytorch_geometric/blob/34c6d5235339872f0d41381f3fbd8700a7d1e8b9/examples/qm9_nn_conv.py#L69).
